src/isomorphic/classic/element/ReactDebugCurrentFrame.js

```diff
@@ -38,7 +38,9 @@ if (__DEV__) {
       if (typeof current === 'number') {
         // DebugID from Stack.
         const debugID = current;
-        stack = getStackAddendumByID(debugID);
+        if (getStackAddendumByID) {
```

Comment:
I'd prefer to not do this since we already know we're in DEV. See my comment below on potential way to accomplish this.